lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -1225,6 +1247,375 @@ struct BroadcastOpConversion
   }
 };
 
+/// ====================== reduce codegen begin ==========================
+
+struct ReduceOpConversion
+    : public ConvertTritonGPUOpToLLVMPattern<triton::ReduceOp> {
+public:
+  using ConvertTritonGPUOpToLLVMPattern<
+      triton::ReduceOp>::ConvertTritonGPUOpToLLVMPattern;
+
+  LogicalResult
+  matchAndRewrite(triton::ReduceOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override;
+
+private:
+  void doAcc(ConversionPatternRewriter &rewriter, Location loc, RedOp redOp,
```

Comment:
accumulate or doAccumulate ? There are not many acronyms in the backend to keep the methods clear to others.

Reply:
`doAcc` was choosed to be consistent with the old name `do_acc` https://github.com/openai/triton/blob/master/lib/codegen/selection/generator.cc#L3131

I agree that full words are easier to understand and I will replace the name `doAcc` with `accumulate`